R/plotIndividualTimeProfile.R

```diff
@@ -0,0 +1,86 @@
+#' Concentration time profile plot
+#'
+#' @param dataCombined A `DataCombined` object.
+#' @param tlfTheme A path to JSON file containing
+#'   [`Theme`](https://www.open-systems-pharmacology.org/TLF-Library/reference/Theme.html)
+#'    object for `{tlf}` library.
+#' @param ospPlotConfiguration A `ospPlotConfiguration` object, which is an `R6`
+#'   class object that defines plot properties (like labels, axes scaling and
+#'   limits, legend position, etc.).
+#' @param xUnit,yUnit Units for x- and y-axes, respectively.
+#'
+#' @import tlf
+#'
+#' @export
```

Comment:
Description of the `tlfTheme `argument is missing

Reply:
You mean there should be more than this? Or you mean there should be another section for theme under the `description` tag?

```
#' @param tlfTheme A path to JSON file containing
#'   [`Theme`](https://www.open-systems-pharmacology.org/TLF-Library/reference/Theme.html)
#'    object for `{tlf}` library.
```

Reply:
I mean I need new glasses - haven't seen the tlfTheme description above 😄